docs/webpack.md

```diff
@@ -155,61 +163,30 @@ fileLoader.exclude = /\.(svg)$/i
 
 ### Url Loader
 
+Be sure to add the default options from the file loader, as those are applied with the file loader if the size is greater than the `limit`.
+
 ```js
-// config/webpack/loaders/url.js
+// config/webpack/environment.js
 
-module.exports = {
+const { environment } = require('@rails/webpacker');
+const rules = environment.loaders;
+
+const urlFileSizeCutover = 10000;
+const urlLoaderOptions = Object.assign({ limit: urlFileSizeCutover }, fileLoader.use[0].options);
+const urlLoader = {
   test: [/\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/],
-  use: [{
+  use: {
     loader: 'url-loader',
-    options: {
-      limit: 10000,
-      name: '[name]-[hash].[ext]'
-    }
-  }]
-}
-
-// config/webpack/environment.js
-
-const { environment } = require('@rails/webpacker')
-const url = require('./loaders/url')
+    options: urlLoaderOptions,
+  },
+};
 
-environment.loaders.prepend('url', url)
+environment.loaders.prepend('url', urlLoader)
 
 // avoid using both file and url loaders
 environment.loaders.get('file').test = /\.(tiff|ico|svg|eot|otf|ttf|woff|woff2)$/i
```

Comment:
this line is not ideal, as it ignores the values set in the config/webpack.yml for static_assets_extensions